includes/Core/Conversion_Tracking/Conversion_Event_Providers/WooCommerce.php

```diff
@@ -195,7 +195,7 @@ public function register_hooks() {
 			function ( $button, $product ) {
 				// If the product is not a valid WC_Product instance, return
 				// early.
-				if ( false === $product instanceof \WC_Product ) {
+				if ( false === $product instanceof \WooCommerce\Abstracts\WC_Product ) {
```

Comment:
This class doesn't seem to exist – `\WC_Product` is the correct FQ class name, we should just import it at the top so we can refer to it safely as `WC_Product` everywhere in the file without the absolute part.